index.html

```diff
@@ -101,7 +142,33 @@ <h3 class="program">Final Session</h3>
             <a class="see" href="https://www.capitaljazz.com/supercruise/2023/">SEE THE WHOLE PROGRAM</a>
         </div>
     </section>
-    <script href="script.js"></script>
+    <article class="singers-card"></article>
```

Comment:
- [ ] Kindly make sure to give this Section a title/header e.g `Speakers`, this will make your page more structured and looks more like the provided design

|Your page|Provided design|
|-|-|
|![Screenshot_20230113_182841](https://user-images.githubusercontent.com/25227022/212382387-22c2ea9f-53ca-44eb-8937-d8d3ad6168ee.png)|![Screenshot_20230113_182803](https://user-images.githubusercontent.com/25227022/212382404-bb530e21-a640-446b-ae7a-c0f9763bb7a4.png)|